engines/gob/inter_playtoons.cpp

```diff
@@ -86,14 +86,14 @@ void Inter_Playtoons::setupOpcodesDraw() {
 
 void Inter_Playtoons::setupOpcodesFunc() {
 	Inter_v6::setupOpcodesFunc();
-	// NOTE: consider backporting here changes done in Inter_v7
-	// in particular, o7_printText (0x0B), o7_drawLine (0x34) and o7_invalidate (0x36)
 
 	CLEAROPCODEFUNC(0x3D);
 	OPCODEFUNC(0x0B, oPlaytoons_printText);
 	OPCODEFUNC(0x1B, oPlaytoons_F_1B);
 	OPCODEFUNC(0x24, oPlaytoons_putPixel);
 	OPCODEFUNC(0x27, oPlaytoons_freeSprite);
+	OPCODEFUNC(0x34, oPlaytoons_drawLine);
+	OPCODEFUNC(0x36, oPlaytoons_invalidate);
```

Comment:
Are you sure about these?

Reply:
Yes, i think so, they are identical. i backported those as the were mentioned in the notes in the code. which is now removed

Reply:
> They are identical
Did you perform some disassembly analysis to support this claim? Or at least some testing showing that the backported opcode fixes a bug in Playtoons for example?